sway-lsp/src/server.rs

```diff
@@ -1037,17 +1140,19 @@ mod tests {
     }
 
     macro_rules! lsp_capability_test {
-        ($test:ident, $capability:expr) => {
+        ($test:ident, $capability:expr, $dir:expr) => {
             #[tokio::test]
             async fn $test() {
-                test_lsp_capability!(doc_comments_dir(), $capability);
+                test_lsp_capability!($dir(), $capability);
             }
         };
     }
 
-    lsp_capability_test!(semantic_tokens, semantic_tokens_request);
-    lsp_capability_test!(document_symbol, document_symbol_request);
-    lsp_capability_test!(format, format_request);
-    lsp_capability_test!(hover, hover_request);
-    lsp_capability_test!(highlight, highlight_request);
+    lsp_capability_test!(semantic_tokens, semantic_tokens_request, doc_comments_dir);
+    lsp_capability_test!(document_symbol, document_symbol_request, doc_comments_dir);
+    lsp_capability_test!(format, format_request, doc_comments_dir);
+    lsp_capability_test!(hover, hover_request, doc_comments_dir);
+    lsp_capability_test!(highlight, highlight_request, doc_comments_dir);
+    lsp_capability_test!(code_action, code_action_request, doc_comments_dir);
+    lsp_capability_test!(code_lens, code_lens_request, e2e_test_dir);
```

Comment:
This one has to be tested using a sway script in order to get any runnables back.

Reply:
Makes sense